### PR TITLE
chore(iOS): Supported new UITableViewStyleInsetGrouped style in table view and list view

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-27171

### CODE REVIEW

File: iphone/Classes/TiUITableView.m [hunk 1]
```diff
@@ -454,7 +454,9 @@ - (UITableView *)tableView
 
     defaultSeparatorInsets = [tableview separatorInset];
 
-    tableview.layoutMargins = UIEdgeInsetsZero;
+    if (style != UITableViewStyleInsetGrouped) {
+      tableview.layoutMargins = UIEdgeInsetsZero;
+    }
```
Comment (hansemannn) [inline iphone/Classes/TiUITableView.m]: Pretty sure this crashes on Xcode 10 and iOS < 13. Maybe:
```objc
#if IS_XCODE_9
if ([TiUtils isiOSVersionOrGreater:@"13.0"] && style != UITableViewStyleInsetGrouped) {
	tableview.layoutMargins = UIEdgeInsetsZero;
}
#else
tableview.layoutMargins = UIEdgeInsetsZero;
#endif
```

Comment (vijaysingh-axway) [reply iphone/Classes/TiUITableView.m]: You are right. Fixed. Thanks!

**Review by janvennemann — APPROVED** [review]
LGTM!

**Comment by keerthi1032** [comment]
FR passed. New UITableViewStyleInsetGrouped style supported on ios 13
 Name                        = Mac OS X
  Version                     = 10.14.5
  Architecture                = 64bit
Node.js
  Node.js Version             = 10.16.2
  npm Version                 = 6.9.0
Titanium CLI
  CLI Version                 = 5.2.1
Titanium SDK
  SDK Version                 = local sdk 8.2.0.v20190718160307
Device - iPhone XR iOS 13
Simulator -iPhone X iOS 13,iphone 6 iOS 12

**Comment by hansemannn** [comment]
@keerthi1032 @vijaysingh-axway This won't be able to merge until unit tests or `np tests` are added.

**Pull request merged**